src/test/java/com/redislabs/redisgraph/RedisGraphAPITest.java

```diff
@@ -881,4 +881,24 @@ public void testArraySupport() {
         }
 
     }
+
+    @Test
+    public void testErrorReporting() {
```

Comment:
Please follow the following example for Juint 4 framework exception assertion:
1. first define a rule
```
@Rule
public ExpectedException exceptionRule = ExpectedException.none();
```
Then, in the test itself, set the rule expected exception type and message:
```
@Test
public void testRuntimeException() {

    api.query("social", "create ()");
    exceptionRule.expect(JedisDataException.class);
    exceptionRule.expectMessage("Type mismatch: expected Integer but was String");
    api.query("social", "RETURN toUpper(5)");
}
```

I know it is a bit awkward, once we'll migrate to Juint 5 it will be easier